templates/getting_started/faq.md

```diff
@@ -289,7 +289,7 @@ It contains the following fields:
 - The default float data type.
 - The default backend. This is legacy; nowadays there is only TensorFlow.
 
-Likewise, cached dataset files, such as those downloaded with [`get_file()`](/utils/#get_file), are stored by default in `$HOME/.keras/datasets/`,
+Likewise, cached dataset files, such as those downloaded with [`get_file()`](https://github.com/keras-team/keras/blob/v2.14.0/keras/utils/data_utils.py#L193), are stored by default in `$HOME/.keras/datasets/`,
```

Comment:
This would pin documentation to version 2.14.0, Keras is currently preparing to release version 3.0.0. Could you change the URL to `/api/utils/python_utils/#get_file-function`.